geomesa-cassandra/geomesa-cassandra-datastore/src/test/resources/cassandra-config.yaml

```diff
@@ -583,4 +584,4 @@ encryption_options:
     # protocol: TLS
     # algorithm: SunX509
     # store_type: JKS
-    # cipher_suites: [TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA]
\ No newline at end of file
+    # cipher_suites: [TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA]
```

Comment:
Is the newline actually part of the improvement?  If not, it looks like nothing substantial may have changed in this file at all, yes?